### PR TITLE
Add WIP radio-at86rf212 driver

### PR DESCRIPTION
**Pull request by ryankurte**

### CODE REVIEW

**Comment by jamesmunns** [comment]
Right.

File: README.md [hunk 1]
```diff
@@ -437,6 +437,7 @@ Work in progress drivers. Help the authors make these crates awesome!
 - [VL53L0X] - A platform agnostic driver to interface with the vl53l0x (time-of-flight sensor) ![crates.io](https://img.shields.io/crates/v/vl53l0x.svg)
 - [MCP49xx] - SPI - 8/10/12-bit DACs like MCP4921, MCP4922, MCP4801, etc. - ![crates.io](https://img.shields.io/crates/v/mcp49xx.svg)
 - [DW1000] - SPI - Radio transceiver for communication based on IEEE 802.15.4 and position tracking - ![crates.io](https://img.shields.io/crates/v/dw1000.svg)
+- [radio-at86rf212] - SPI - Sub GHZ 802.15.4 radio transceiver ![crates.io](https://img.shields.io/crates/v/radio-at86rf212.svg)
```
Comment (therealprof) [inline README.md]:
```suggestion
- [radio-at86rf212] - SPI - Sub GHz 802.15.4 radio transceiver ![crates.io](https://img.shields.io/crates/v/radio-at86rf212.svg)
```

**Comment by berkus** [comment]
what is going on, i though we don't use bors in this repository?

**Comment by therealprof** [comment]
True, my bad. I'm so used to using bors when I see the badge that I forgot that we merge manually in here. ;)

**Pull request merged**